website/docs/reference/administrative-commands/createIndexes.mdx

```diff
@@ -0,0 +1,179 @@
+| `partialFilterExpression` | document | Restricts index to documents matching the condition.                                                                                            |
+| `sparse`                  | boolean  | Only indexes documents that contain the indexed field if set to `true`.                                                                         |
+
+## Example usage
+
+The following examples demonstrate how to create different types of indexes.
+
+### Simple index
+
+The simplest way to create an index is to use the `createIndexes` command with the `key` field (the field(s) to be indexed) and their sort order (`1` for ascending, `-1` for descending).
+
+<!-- Syntax structure of the command type, only included here within subcategories-->
+
+```js
+db.runCommand({
+  createIndexes: '<collection_name>',
+  indexes: [
+    {
+      key: {
+        '<field1>': 1
+      },
+      name: '<index_name>'
+    }
+  ]
+})
+```
+
+This example creates a simple index on the `title` field in the `books` collection.
+
+<CodeBlock language="js">{CreateIndex}</CodeBlock>
+
+### Compound index
+
+Compound indexes enables multiple fields to be indexed together, as shown in the syntax below:
+
+```js
+db.runCommand({
+  createIndexes: '<collection_name>',
+  indexes: [
+    {
+      key: {
+        '<field1>': 1,
+        '<field2>': -1
+      },
+      name: '<compound_index_name>'
+    }
+  ]
+})
+```
+
+The example below creates a compound index on the `publication.date` and `analytics.average_rating` fields in the `books` collection.
+
+<CodeBlock language="js">{CreateCompoundIndex}</CodeBlock>
+
+### Unique index
+
+Unique indexes ensure that your indexed fields only contain unique values, without duplicates.
+
+```js
+db.runCommand({
+  createIndexes: '<collection_name>',
+  indexes: [
+    {
+      key: {
+        '<field>': 1
+      },
+      name: '<unique_index_name>',
+      unique: true
+    }
+  ]
+})
+```
+
+This example creates a unique index on the `isbn` field in the `books` collection to ensure that each book has a unique ISBN.
+
+<CodeBlock language="js">{CreateUniqueIndex}</CodeBlock>
+
+### Partial index
+
+A partial index is similar to a regular index, however, it only indexes a fraction of documents in a collection based on a specified filter condition.
+
+```js
+db.runCommand({
+  createIndexes: '<collection_name>',
+  indexes: [
+    {
+      key: {
+        '<field>': 1
+      },
+      name: '<partial_index_name>',
+      partialFilterExpression: {
+        '<field>': { '<operator>': '<value>' }
+      }
+    }
+  ]
+})
+```
+
+Let's create a partial index on the `books` collection only for books published after the year 2000.
+
+<CodeBlock language="js">{CreatePartialIndex}</CodeBlock>
+
+### Sparse index
+
+Sparse indexes only sets an index on documents with the specified field, even if the field is not present in all documents.
```

Comment:
I think we need to clarify the description. For normal index, if the field doesn't exist, the document is still indexed. For sparse index, if the field doesn't exist, that document is not indexed. Am I correct?